### PR TITLE
New Binomial unit test

### LINKED ISSUE #2552: ComputeQuantile for discrete variables
### What happened?

We have a discrete variable, and we want to calculate its quantile of order p.
The two following methods should be identical: 
computeQuantile(p) and computeQuantile(1-p, True).
But the results obtained differ....

### How to reproduce the issue?

```shell
from openturns import *

n=16
alpha = 0.75
bin = Binomial(n, alpha)

proba = 0.95
q1 = bin.computeQuantile(proba)[0]
q2 = bin.computeQuantile(1-proba, True)[0]
print('q1, q2 = ', q1, q2)

# Test q1: 
print(bin.computeCDF(q1))
# 0.9899774042423815 > proba ==> ok!
# Test q2: 
print(bin.computeCDF(q2))
# 0.9365235602017492< proba ==> problem!
```

### Version

1.23dev

### Operating System

Linux

### Installation media

from source

### Additional Context

_No response_

### PR DESCRIPTION
**Pull request by mbaudin47**
As a follow-up to PR #2046 and in order to close issue #2552 _forever_ (hopefully :)), this PR implements a unit test of the Binomial distribution using a CSV dataset which is already in the unit tests, but unused.
This test currently fails, because of issue #2552. It is expected to pass once PR #2687 is merged.

### CODE REVIEW

**Review by regislebrun — APPROVED** [review]

**Review by josephmure — APPROVED** [review]

**Comment by josephmure** [comment]
Hi @mbaudin47 have you had the time to look at this again? Apparently the tests still fail.

**Comment by mbaudin47** [comment]
@jschueller : This PR is ready from my point of view.

File: python/test/t_Binomial_std.py [hunk 1]
```diff
@@ -64,6 +68,55 @@
...
 assert distribution.computeScalarQuantile(0.9, True) == 0
 
+# %%
+print("Check on dataset")
+separator = ","
+skipped_lines = 13
+path = os.path.join(os.path.dirname(__file__), "t_binomial_dataset.csv")
+sample = ot.Sample.ImportFromTextFile(path, separator, skipped_lines)
+rtol = 1.e-12
+atol = 1.e-300
```
Comment (jschueller) [inline python/test/t_Binomial_std.py:78]: MinScalar ?

File: python/test/t_Binomial_std.py [hunk 2]
```diff
@@ -64,6 +68,55 @@
 distribution = ot.Binomial(3, 0.5)
 assert distribution.computeScalarQuantile(0.9, True) == 0
 
+# %%
+print("Check on dataset")
+separator = ","
+skipped_lines = 13
+path = os.path.join(os.path.dirname(__file__), "t_binomial_dataset.csv")
```
Comment (jschueller) [inline python/test/t_Binomial_std.py:75]: I would rename it to t_Binomial_std.csv so it has the same name as the test

File: python/test/t_Binomial_std.py [hunk 3]
```diff
@@ -64,6 +68,55 @@
...
+        # The function is not invertible
+        # for this particular input.
+        continue
+    elif expected_cdfp < expected_cdfq:
+        computed_x = int(distribution.computeQuantile(computed_cdf)[0])
+        cdfXM1 = distribution.computeCDF(computed_x - 1)
+        cdfX = distribution.computeCDF(computed_x)
+        print(f"    computeQuantile (A). Computed X = {computed_x}, F(X - 1) = {cdfXM1:.7e}, F(X) = {cdfX:.7e}")
+        if computed_cdf == 0.0:
+            assert computed_x == 0.0
```
Comment (jschueller) [inline python/test/t_Binomial_std.py:110]: redundant

File: python/test/t_Binomial_std.py (hunk 1 above)
Comment (mbaudin47) [reply python/test/t_Binomial_std.py:78]: This is true: I fix this.

**Pull request merged**